### ISSUE TITLE
🐛 Bug Report: Gitlab plugin not loading

### ISSUE THREAD

**Issue by matiri132** [backstage/backstage#15214]
### 📜 Description

After upgrade backstage to last version (1.8.3 from 1.5.0), i got the entity page of Gitlab  plugin not working.

### 👍 Expected behavior

Keep gitlab plugin working properly

### 👎 Actual Behavior with Screenshots

![image](https://user-images.githubusercontent.com/26864384/207400922-4082aa34-dc33-4847-8eef-672b7caf332c.png)

### 👟 Reproduction steps

1. Go to Gitlab tab in the entity page

### 📃 Provide the context for the Bug.

Before the upgrade the plugins was working ok.

packages/app/package.json
```
{
  "name": "app",
  "version": "0.0.1",
  "private": true,
  "bundled": true,
  "backstage": {
    "role": "frontend"
  },
  "dependencies": {
    "@backstage/app-defaults": "^1.0.8",
    "@backstage/catalog-model": "^1.1.3",
    "@backstage/cli": "^0.21.0",
    "@backstage/core-app-api": "^1.2.0",
    "@backstage/core-components": "^0.12.0",
    "@backstage/core-plugin-api": "^1.1.0",
    "@backstage/integration-react": "^1.1.6",
    "@backstage/plugin-api-docs": "^0.8.11",
    "@backstage/plugin-catalog": "^1.6.1",
    "@backstage/plugin-catalog-common": "^1.0.8",
    "@backstage/plugin-catalog-graph": "^0.2.23",
    "@backstage/plugin-catalog-import": "^0.9.1",
    "@backstage/plugin-catalog-react": "^1.2.1",
    "@backstage/plugin-github-actions": "^0.5.11",
    "@backstage/plugin-org": "^0.6.0",
    "@backstage/plugin-permission-react": "^0.4.7",
    "@backstage/plugin-playlist": "^0.1.2",
    "@backstage/plugin-scaffolder": "^1.8.0",
    "@backstage/plugin-search": "^1.0.4",
    "@backstage/plugin-search-react": "^1.2.1",
    "@backstage/plugin-sentry": "^0.4.4",
    "@backstage/plugin-tech-radar": "^0.5.18",
    "@backstage/plugin-techdocs": "^1.4.0",
    "@backstage/plugin-techdocs-module-addons-contrib": "^1.0.6",
    "@backstage/plugin-techdocs-react": "^1.0.6",
    "@backstage/plugin-user-settings": "^0.5.1",
    "@backstage/theme": "^0.2.16",
    "@loblaw/backstage-plugin-gitlab": "^0.1.24",
    "@material-ui/core": "^4.12.2",
    "@material-ui/icons": "^4.9.1",
    "@roadiehq/backstage-plugin-argo-cd": "^2.2.0",
    "@roadiehq/backstage-plugin-datadog": "^2.1.0",
    "history": "^5.0.0",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-router": "^6.3.0",
    "react-router-dom": "^6.3.0",
    "react-use": "^17.2.4"
  },
  "devDependencies": {
    "@backstage/test-utils": "^1.2.2",
    "@testing-library/jest-dom": "^5.10.1",
    "@testing-library/react": "^12.1.3",
    "@testing-library/user-event": "^14.0.0",
    "@types/jest": "^26.0.7",
    "@types/node": "^16.11.26",
    "@types/react-dom": "*",
    "cross-env": "^7.0.0",
    "cypress": "^9.7.0",
    "eslint-plugin-cypress": "^2.10.3",
    "start-server-and-test": "^1.10.11"
  },
  "scripts": {
    "start": "backstage-cli package start",
    "build": "backstage-cli package build",
    "clean": "backstage-cli package clean",
    "test": "backstage-cli package test",
    "lint": "backstage-cli package lint",
    "test:e2e": "cross-env PORT=3001 start-server-and-test start http://localhost:3001 cy:dev",
    "test:e2e:ci": "cross-env PORT=3001 start-server-and-test start http://localhost:3001 cy:run",
    "cy:dev": "cypress open",
    "cy:run": "cypress run --browser chrome"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "files": [
    "dist"
  ]
}
```

### 🖥️ Your Environment

Backstage running in Kubernetes.

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

No, but I'm happy to collaborate on a PR with someone else

**Comment by sbarrypoppulo** [comment]
Related to https://github.com/loblaw-sre/backstage-plugin-gitlab/issues/230. The third-party plugin developer needs to update for the [React Router v6 migration](https://backstage.io/docs/tutorials/react-router-stable-migration#for-plugin-authors).

Note a PR exists on the repo but hasn't been merged - https://github.com/loblaw-sre/backstage-plugin-gitlab/pull/190

**Comment by matiri132** [comment]
> Related to [loblaw-sre/backstage-plugin-gitlab#230](https://github.com/loblaw-sre/backstage-plugin-gitlab/issues/230). The third-party plugin developer needs to update for the [React Router v6 migration](https://backstage.io/docs/tutorials/react-router-stable-migration#for-plugin-authors).
> 
> Note a PR exists on the repo but hasn't been merged - [loblaw-sre/backstage-plugin-gitlab#190](https://github.com/loblaw-sre/backstage-plugin-gitlab/pull/190)

Solved following the PR described by @sbarrypoppulo
